### PR TITLE
added the pages of the app clone

### PR DESCRIPTION
**Pull request by knee-rel**

### CODE REVIEW

File: components/Carousel.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+export default function Carousel() {
```
Comment (ayabbear) [inline components/Carousel.js:3]: Tama naman yung ginawa mo with
```
export default function Carousel() {
    return ()
}
```
tho yung saten the way we declare functions, para may uniformity lang, is we use arrow functions and default export like this
```
const Carousel = () => {
    return ()
}

export default Carousel
```

File: components/Carousel.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    return (
+        <div className = 'carousel'>
+                <div className='carousel-inner'>
+                    <input className='carousel-open' type="radio" id="carousel-1" name="carousel" aria-hidden="true" hidden="" checked="checked"/>
+                    <div className='carousel-item'>
+                        <img src ='../assets/modal-1.png' />
+                    </div>
+                    <input className='carousel-open' type="radio" id="carousel-1" name="carousel" aria-hidden="true" hidden="" checked="checked"/>
+                    <div className='carousel-item'>
+                        <img src ='../assets/modal-1.png' />
```
Comment (ayabbear) [inline components/Carousel.js:13]: `` naka-save ba to sa `public` na directory? if so, create a new directory sa loob ng `src` for the images, oks lang naman sya but this is how we usually do this

File: components/Carousel.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+                    </div>
+                    <label for="carousel-3" class="carousel-control prev control-1">‹</label>
+                    <label for="carousel-2" class="carousel-control next control-1">›</label>
+                    <label for="carousel-1" class="carousel-control prev control-2">‹</label>
+                    <label for="carousel-3" class="carousel-control next control-2">›</label>
+                    <label for="carousel-2" class="carousel-control prev control-3">‹</label>
+                    <label for="carousel-1" class="carousel-control next control-3">›</label>
+                    <ol class="carousel-indicators">
+                        <li>
+                            <label for="carousel-1" class="carousel-bullet">•</label>
```
Comment (ayabbear) [inline components/Carousel.js:23]: for codes na paulit ulit and nagbabago lang ng value, you can use `map` to iterate and `string interpolation` to change yung values ng string using a variable, for this example
```
<li>
    <label for="carousel-1" class="carousel-bullet">•</label>
</li>
<li>
    <label for="carousel-2" class="carousel-bullet">•</label>
</li>
<li>
    <label for="carousel-3" class="carousel-bullet">•</label>
</li>
```
first you create an object/array for your data
`const labels = ['1','2','3']`

then you do mapping
```
labels.map((value,key) => {
    return (
    <li key={key}>
        <label for=`carousel-${value}` class="carousel-bullet">•</label>
    </li>
    )
})
```
you can do `string interpolation` like this ``` `im a string ${yourVariable}` ```

File: components/Dresses.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    return(
+        <main class='my-8'>
+            <div className='container mx-auto px-6'>        
+                <h3 className='text-gray-700 text-2xl font-medium'>Dresses</h3>
+                <span className='mt-3 text-sm text-gray-500'>60+ Results</span>   
+                <div className='grid gap-6 grid-cols-1 sm:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4 mt-6'>
+                    <div className='w-full max-w-sm mx-auto rounded-md shadow-md overflow-hidden'>
+                        <div className = 'flex items-end justify-end h-56 w-full bg-cover'>
+                            <button className='p-2 rounded-full bg-blue-600 text-white mx-5 -mb-4 hover:bg-blue-500 focus:outline-none focus:bg-blue-500'>
+                                <svg className="h-5 w-5" fill="none" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" viewBox="0 0 24 24" stroke="currentColor"><path d="M3 3h2l.4 2M7 13h10l4-8H5.4M7 13L5.4 5M7 13l-2.293 2.293c-.63.63-.184 1.707.707 1.707H17m0 0a2 2 0 100 4 2 2 0 000-4zm-8 2a2 2 0 11-4 0 2 2 0 014 0z"></path></svg>
```
Comment (ayabbear) [inline components/Dresses.js:13]: for `svg`'s, you can use `fontawesome`, kahit yung CDN nalang nila

File: components/Dresses.js [hunk 5]
```diff
@@ -0,0 +1,87 @@
+import React from 'react';
```
Comment (ayabbear) [inline components/Dresses.js:1]: it looks like same codes lang yung ginamit mo with your categories of `Dresses`, `Eyewear`, etc.. you can do the `dynamic rendering` / `dynamic page` sa loob ng `pages` directory, you can do something like this
```
pages
--categories
----[category]
```
then para makapunta ka sakanya from the `button`  you use `Link` like this
```
import Link from 'next/link'

const category = 'Dresses'

<Link href={{ pathname: `/categories/${category }` }}>
   <button>I'm a button</button>
</Link>
```

File: components/Navbar.js [hunk 6]
```diff
@@ -9,7 +9,7 @@ export default function Navbar() {
           <div className="w-full relative flex justify-between lg:w-auto lg:static lg:block lg:justify-start">
             <a
               className="text-sm font-bold leading-relaxed inline-block mr-4 py-2 whitespace-no-wrap uppercase text-white"
-              href="#pablo"
+              href="/"
```
Comment (ayabbear) [inline components/Navbar.js:12]: for routing pages use `Link` from Next.js

Comment (knee-rel) [reply components/Navbar.js:12]: done

**Pull request closed without merging**